inline.go

```diff
@@ -638,23 +638,33 @@ func stripMailto(link []byte) []byte {
 	}
 }
 
+// autoLinkType specifies a kind of autolink that gets detected.
+type autoLinkType int
+
+// These are the possible flag values for the autolink renderer.
+// Only a single one of these values will be used; they are not ORed together.
```

Comment:
```Go
// Only a single one of these values will be used; they are not ORed together.
```

I would leave this sentence out. That should be considered as the default for enums. Saying it explicitly makes sense if it weren't the case.